Formula/llvm.rb

```diff
@@ -381,8 +402,21 @@ def caveats
               "-I#{MacOS.sdk_path}/usr/include",
               "-L#{lib}",
               "-Wl,-rpath,#{lib}", "test.cpp", "-o", "test"
-      assert_includes MachO::Tools.dylibs("test"), "#{opt_lib}/libc++.1.dylib"
+      assert_includes MachO::Tools.dylibs("test"), "@rpath/libc++.1.dylib"
       assert_equal "Hello World!", shell_output("./test").chomp
+
+      # link against installed libc++experimental
+      if build.with? "experimental"
+        system "#{bin}/clang++", "-v", "-nostdinc",
+                "-std=c++11", "-stdlib=libc++",
+                "-lc++experimental",
+                "-I#{opt_include}/c++/v1",
+                "-I#{libclangxc}/include",
+                "-I#{MacOS.sdk_path}/usr/include",
+                "-L#{lib}",
+                "-Wl,-rpath,#{lib}", "test_experimental.cpp", "-o", "test_experimental"
+        assert_equal "\"#{Pathname.pwd}\"", shell_output("./test_experimental").chomp
```

Comment:
If Homebrew breaks the `testpath` helper there's going to be a whole world of pain beyond this formula, heh. But yes, unless you're moving to a new directory deliberately inside `test do` you should be in `testpath`.